File: Adapter/Product/EnemyRobot.cs
using System;

namespace Adapter.Product {
    internal class EnemyRobot {
        Random random = new Random();

        public void SlashWithSword() {
            int attackDamage = random.Next(10) + 1;
            Console.WriteLine("Enemy Robot slashes " + attackDamage + " damages with sword");
        }

        public void WalkForward() {
            int movement = random.Next(5) + 1;
            Console.WriteLine("Enemy Robot walks forward " + movement + " spaces");
        }

        public void ReactToHuman(String driverName) {
            Console.WriteLine(driverName + " is controlling the robot");
        }
    }
}

File: Adapter.java/src/com/company/EnemyRobot.java
package com.company;

import java.util.Random;

public class EnemyRobot {
    Random random = new Random();

    public void slashWithSword() {
        int attackDamage = random.nextInt(10) + 1;
        System.out.println("Enemy Robot slashes " + attackDamage + " damages with sword");
    }

    public void walkForward(){
        int movement = random.nextInt(5) + 1;
        System.out.println("Enemy Robot walks forward " + movement + " spaces");
    }

    public void reactToHuman(String driverName) {
        System.out.println(driverName + " is controlling the robot");
    }

}
